### PR TITLE
Fix the alignment of the wallet background

### LINKED ISSUE #2826: Fix the alignment of the wallet background
How it looks
Screenshot 2021-10-29 at 19 05 36

How it should look (1 px around the darker background)
Screenshot 2021-10-29 at 19 11 40

### PR DESCRIPTION
**Pull request by arrenv**
## Description

Update wallet background alignment to match design.

**Changes** 🏗

* Changed styling to utilise CSS Flexbox

8815eae2-96d7-4fa8-bdd7-asd24f0b0

Resolves  #2826

### CODE REVIEW

File: src/modules/pages/components/RouteLayouts/UserNavigation.css [hunk 1]
```diff
@@ -138,7 +138,12 @@
 }
 
 .buttonsWrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
   margin-right: 12px;
+  padding: 1px;
```
Comment (chinins) [inline src/modules/pages/components/RouteLayouts/UserNavigation.css:146]: Can you please contact Karol & confirm with him if it should be 1px or 2px. I think that it's 1 px but just to make sure.

**Review by chinins — COMMENTED** [review]
It's all looking good on my machine & with different zoom options too. Just please, confirm with Karol about the padding to make sure it's 1px.

As I mentioned during the call, the test will also be if it renders ok on Raul's & Armando's machines.

**Review by ArmandoGraterol — COMMENTED** [review]
To me, the padding looks right as the one in the design, however, the wallet address isn't aligned with the token display:

FireShot Capture 465 - Colony - localhost

I think the wallet address should be 1px or 2px more to the bottom to be correctly aligned

File: src/modules/pages/components/RouteLayouts/UserNavigation.css (hunk 1 above)
Comment (arrenv) [reply src/modules/pages/components/RouteLayouts/UserNavigation.css:145]: Thank you, I have reached out to confirm. I have added a PR in the meantime, which should fix slight misalignment of text.

**Comment by arrenv** [comment]
> To me, the padding looks right as the one in the design, however, the wallet address isn't aligned with the token display:
> 
> FireShot Capture 465 - Colony - localhost
> 
> I think the wallet address should be 1px or 2px more to the bottom to be correctly aligned

Thank you Armando, I have removed the line-height property, which should correct this. However, let me know if it is improved on your end. I have noticed a misaligning across the other components (see screenshot). Pink line shows misaligned, green shows exact top and bottom margin.

header-alignment-outline

File: src/modules/pages/components/RouteLayouts/UserNavigation.css (hunk 1 above)
Comment (arrenv) [reply src/modules/pages/components/RouteLayouts/UserNavigation.css:145]: Thank you, confirmed with Karol that it is 1px.

**Review by ArmandoGraterol — APPROVED** [review]
Looks good to me! Now it looks much better :)
FireShot Capture 466 - Colony - localhost

File: src/modules/pages/components/RouteLayouts/UserNavigation.css [hunk 2]
```diff
@@ -132,13 +120,14 @@
 
 .walletAddressActive {
   composes: walletAddress;
-  padding: 1px 8px 2px;
   border: 1px solid var(--colony-blue);
   color: var(--colony-blue);
 }
 
 .buttonsWrapper {
+  composes: flexContainerRow flexAlignCenter flexJustifyCenter from '~styles/layout.css';
```
Comment (rdig) [inline src/modules/pages/components/RouteLayouts/UserNavigation.css:128]: :+1:

**Review by rdig — APPROVED** [review]
Very nice @arrenv :100: Looks much better now!

Screenshot from 2021-11-09 19-40-54

_Just a heads up that I've pushed some changes to `master` and you'll have to rebase before you'll be able to merge_

**Comment by chinins** [comment]
let me know once you rebase - i will check the UI on my machine as well (I don't want to rebuild the environment 🤓)

**Comment by arrenv** [comment]
> let me know once you rebase - i will check the UI on my machine as well (I don't want to rebuild the environment 🤓)

Sure thing, I have rebased now.

**Comment by chinins** [comment]
Looks good on my machine too - well done!

**Pull request merged**